nix/shared.nix

```diff
@@ -101,6 +101,12 @@ let
 
       twitter = pkgsNew.callPackage ./twitterLogo.nix { };
 
+      xml =
+        pkgsNew.fetchurl {
+          url    = "https://upload.wikimedia.org/wikipedia/commons/9/9d/Xml_logo.svg";
+          sha256 = "15a9a1ga94l9x1ns759nn8cmvp4x0m634c492ilcv97p1vmx6wj7";
```

Comment:
After some searching, I created the hash from the downloaded file with

    nix-hash --base32 --flat --type sha256 xml.svg

Would there have been an easier way?

Reply:
@sjakobi: `nix-prefetch-url` is the easier way to do this